nbs/api/export.ipynb

```diff
@@ -134,7 +134,26 @@
    "outputs": [],
```

Comment:
<p>Please note that the implementation is just copying <code>processors.clean_magics</code>. Not sure how much of an issue that is for this repository. Should we add a code cell like below above this <code>export.scrub_magics</code> method?</p><p> </p><p> </p><p><code>\#|hide</code></p><p><code>\# FIXME: reusing processors.clean_magics runs into a circular import</code></p><p><code>\# from .processors import clean_magics<span class="ql-cursor">﻿</span></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>